test/e2e/index.test.ts

```diff
@@ -189,7 +189,7 @@ async function initializeEnvironment(
         await waitFor(async () => {
           expect(await page.evaluate(() => document.activeElement?.textContent)).to.equal('100');
           expect(await page.evaluate(() => document.activeElement?.getAttribute('role'))).to.equal(
-            'button',
+            'combobox',
```

Comment:
The role of the `Select` component has been changed from `button` to `combobox` with https://github.com/mui/material-ui/pull/38785
cc @mui/xgrid